spec/models/photo_spec.rb

```diff
@@ -45,8 +45,7 @@ def with_carrierwave_processing(&block)
   describe '#diaspora_initialize' do
     before do
       @image = File.open(@fixture_name)
-      @photo = Photo.diaspora_initialize(
-                :author => @user.person, :user_file => @image)
+      @photo = Photo.diaspora_initialize(:author => @user.person, :user_file => @image)
```

Comment:
Please update to Ruby 1.9 hash syntax while touching: `author: @user.person, user_file: @image`